skins/laika/src/components/pages/donation_form/Email.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-    <div>
-		<label for="email" class="subtitle has-margin-top-18">{{ $t( 'email_label' ) }}</label>
+    <fieldset class="has-margin-top-36">
+		<h1 class="title is-size-1">{{ $t( 'donation_section_email_title' ) }}</h1>
```

Comment:
the email title is in `Address.vue` on purpose, to make `Email.vue` reusable. Pls put it back

Reply:
Ouf, it breaks the design because the title is just randomly floating in between the div elements. I will try and come up with an alternative.

Reply:
@tzhelyazkova Should be good now, I hope.